### PR TITLE
Begin update for 2.3-3.0 blog

### PR DESCRIPTION
**Pull request by FionaDL**
Hey this PR begins to add some things to the blog about upgrading from 2.3 => 3.0. It would be great if @arielj or @rdormer could also check out this article and see if they have things to add. I will add this as WIP for now. 

https://www.pivotaltracker.com/story/show/175584267

### CODE REVIEW

File: _posts/2017-08-28-upgrade-to-rails-3.markdown [hunk 1]
```diff
@@ -147,6 +148,24 @@ end
...
+```
+link_to_remote ("Update Example", 
+                        :update => 'example',
+                        :url => {:action => 'example'})
+```
+Will need to change to:
+```
+link_to "Update Example", { :action => 'example' }, remote => true 
+```
+and unobtrusive JavaScript will need to be added for the what "update" is doing.
```
Comment (arielj) [inline _posts/2017-08-28-upgrade-to-rails-3.markdown:165]:
```suggestion
and a JavaScript event listener will need to be added for what "update" is doing.
```

Maybe we can add a small code snippet explaining this line? no too fancy, just the `$(...).on('ajax:success', function(response) {.... $('#example').html(response.responseText)}` or something like that. The link below is not working for me (maybe this is shown there but I think it's better to have an example on our article)

**Review by arielj — APPROVED** [review]
I approved it, but don't merge it yet, I will add some comments to the post next week

**Comment by arielj** [comment]
@FionaDL can you review my changes too?

@lubc this should be ready if fiona and you approve my changes

File: _posts/2017-08-28-upgrade-to-rails-3.markdown [hunk 2]
```diff
@@ -78,15 +83,44 @@ end
 
 Remember that if you installed the plugin mentioned in step 3, you can run `rake rails:upgrade:gems`. This task will extract your config.gem calls and generate code that you can put in your Gemfile.
 
-<h2 id="deprecations">6. Deprecations</h2>
+<h2 id="rspec">7. RSpec</h2>
+If you are using RSpec 1.x for your tests, you should update to RSpec 2.x. You may need to update some references to `Spec` with `RSpec`.
+
+<h2 id="ignoring">8. Ignoring Rails Modules</h2>
+If you don't need to load some module (let's use actionmailer as an example), in Rails 2 you would a configuration like this in your environments.rb file:
```
Comment (FionaDL) [inline _posts/2017-08-28-upgrade-to-rails-3.markdown:90]:
```suggestion
If you don't need to load a module (let's use actionmailer as an example), in Rails 2 you would use a configuration like this in your environments.rb file:
```

**Review by FionaDL — COMMENTED** [review]
@arielj one small comment. Also, I think we should a dd a part about html_safe, maybe we can look at that together.

File: _posts/2017-08-28-upgrade-to-rails-3.markdown [hunk 3]
```diff
@@ -78,15 +84,46 @@ end
...
+  rails/test_unit
+).each do |framework|
+  begin
+    require "#{framework}/railtie"
+  rescue LoadError
+  end
+end
+```
+<h2 id="html-safe">9. Escaped HTML by default</h2>
+In Rails 3 you no  longer need to use the `h` helper method to escape HTML. It is now escaped by default. If you need to tell Rails to not escape the HTML you will need to call `html_safe` on that string.
```
Comment (arielj) [inline _posts/2017-08-28-upgrade-to-rails-3.markdown:116]:
```suggestion
In Rails 3 you no longer need to use the `h` helper method to escape HTML. It is now escaped by default. If you need to tell Rails to not escape the HTML you will need to call `.html_safe` on that string. 
```

**Comment by FionaDL** [comment]
@lubc I think we are ready for a review here.

File: _posts/2017-08-28-upgrade-to-rails-3.markdown [hunk 4]
```diff
@@ -147,6 +184,32 @@ end
...
+                :update => 'result_div_id',
+                :url => {:action => 'example'})
+```
+
+Will need to change to:
+
+```js
+link_to "Update Example", { :action => 'example' }, :remote => true, :id => 'my_link'
+```
+and a JavaScript event listener will need to be added for what "update" is doing.
```
Comment (lubc) [inline _posts/2017-08-28-upgrade-to-rails-3.markdown:204]:
```suggestion
And a JavaScript event listener will need to be added for what "update" is doing.
```

**Review by lubc — APPROVED** [review]
@FionaDL @arielj Looks good👍  Thanks!

**Pull request merged**